include/aspect/postprocess/particle_crystal_preferred_orientation.h

```diff
@@ -0,0 +1,322 @@
+/*
+ Copyright (C) 2022 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef _aspect_postprocess_particle_crystal_preferred_orientation_h
```

Comment:
Shouldnt "crystal_preferred_orientation" be enough as a name for the postprocessor? "particle" just happens to be the method we are using at the moment.

Reply:
I did that to indicate that it build on the particle plugin. I will change it.